assets/src/containers/SideDrawer.js

```diff
@@ -30,11 +32,19 @@ function SideDrawer (props) {
 
   const courseId = location.pathname.split('/')[1]
   const [selectedIndex, setSelectedIndex] = useState(false)
+  const [loaded, courseInfo] = useCourseInfo(courseId)
+  const [activeCourseViews, setActiveCourseViews] = useState({})
+
+  useEffect(() => {
+    if (loaded) {
+      setActiveCourseViews(courseInfo.course_view_options)
```

Comment:
The code looks good to me. 

One thing to be noticed is that the current cards displayed on the index page are always left aligned, with a fix-width center-aligned flex wrapper. It looks a little bit odd when the number of cards is less than 3 since all the cards are left-aligned. I'm thinking of using the number of cards to dynamically change the maxWidth of the wrapper element. Something like:

```
const cardWidth = 340;
const [numOfCard, setNumOfCard] = useState(0)

useEffect(() => {
    if (loaded) {
      setActiveCourseViews(courseInfo.course_view_options)
      let cnt = 0;
      for (let option of Object.keys(courseInfo.course_view_options)) {
        if (courseInfo.course_view_options[option] === 1) {
          cnt ++;
        }
      }
      setNumOfCard(Math.min(3, cnt));
    }
  }, [loaded])

...
<Grid
   container
   className={classes.wrapper}
   style = {{maxWidth: numOfCard * cardWidth + "px"}}
   spacing={8}>
```

It's a little bit hacky so feel free to suggest any other possible solutions for this styling problem. I can open another PR for this if needed.

Reply:
I think it's okay for it to be left aligned. The code above makes the cards 'stretch' out if there's fewer than 3?

Reply:
No, the above code will make sure the cards are center-aligned even when there're fewer than 3 cards. The problem is that if there are only one or two cards, they'll not be centered on the page by current styling. Because they are wrapped in a left-aligned flex box.

Reply:
Ah okay. I personally think left-aligned is okay.